### PR TITLE
French translations

### PR DESCRIPTION
**Pull request by eggrobin**
Also changes to the celestial strings for apsides that should have been in #3101.

### CODE REVIEW

File: ksp_plugin_adapter/localization/fr-fr.cfg [hunk 1]
```diff
@@ -0,0 +1,241 @@
+Localization {
+  fr-fr {
+    // Placeholder-only strings for extracting specific forms.
+    #Principia_GrammaticalForm_Standalone = <<C:1>>
+
+    // MainWindow
+    #Principia_MainWindow_PluginNotStarted = Plugin non démarré
+    // <<1>> next_release_lunation_number <<2>> next release name
+    #Principia_MainWindow_NewMoonAnnouncementWithKspDeprecation = Avertissement: la nouvelle lune de la lunaison numéro <<1>> est arrivée ; merci de mettre à jour KSP à la version <<3>> et de télécharger la dernière version de Principia, <<2>>. Notez que RealismOverhaul et RealSolarSystem supportent maintenant KSP <<3>>.
```
Comment (pleroy) [inline ksp_plugin_adapter/localization/fr-fr.cfg:9]: Space before `:`.

File: ksp_plugin_adapter/localization/fr-fr.cfg [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+    #Principia_MainWindow_NewMoonAnnouncement = Avertissement: la nouvelle lune de la lunaison numéro <<1>> est arrivée ; merci de télécharger la dernière version de Principia, <<2>>.
```
Comment (pleroy) [inline ksp_plugin_adapter/localization/fr-fr.cfg:10]: Same.

File: ksp_plugin_adapter/localization/fr-fr.cfg [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+    #Principia_MainWindow_TargetVessel_Select = Sélectionner le vaisseau cible...
+    #Principia_MainWindow_TargetVessel_Name = Cible: <<1>>  // <<1>> vessel name
+    #Principia_MainWindow_TargetVessel_Clear = Effacer
+    #Principia_MainWindow_TargetVessel_SwitchTo = Échanger
+    #Principia_MainWindow_PredictionSettings = Paramètres de prédiction
+    #Principia_MainWindow_KspFeatures = Fonctionalités de KSP
+    #Principia_MainWindow_LoggingSettings = Paramètres de logging
+    #Principia_MainWindow_KspFeature_DisplayPatchedConics = Afficher les coniques juxtaposées (ne pas utiliser pour les plans de vol !)
+    #Principia_MainWindow_TargetCelestial_Select = Sélectionner le corps cible...
+    #Principia_MainWindow_TargetCelestial_Name = Cible: <<A:1>>  // <<1>> celestial name
```
Comment (pleroy) [inline ksp_plugin_adapter/localization/fr-fr.cfg:20]: Same.

File: ksp_plugin_adapter/localization/fr-fr.cfg [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+    #Principia_MainWindow_TargetCelestial_Clear = Effacer
+    #Principia_MainWindow_LoggingSettings_VerboseLevel = Niveau de verbosité :
+    #Principia_MainWindow_LoggingSettings_LogOption = Traces
+    #Principia_MainWindow_LoggingSettings_StderrOption = stderr
+    #Principia_MainWindow_LoggingSettings_FlushOption = Immédiat
+    #Principia_MainWindow_LoggingSettings_RecordJournal = Enregistrer un journal (démarre au chargement)
+    #Principia_MainWindow_LoggingSettings_RecordJournalResult = Le journal est <<1>>  // <<1>> ON/OFF
+    #Principia_MainWindow_LoggingSettings_JournalingStatus_ON = ACTIF
+    #Principia_MainWindow_LoggingSettings_JournalingStatus_OFF = INACTIF
+    #Principia_PredictionSettings_ToleranceLabel = Tolérance:
```
Comment (pleroy) [inline ksp_plugin_adapter/localization/fr-fr.cfg:30]: Same.

File: ksp_plugin_adapter/localization/fr-fr.cfg [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+    // ReferenceFrameSelector
+
+    // For the names, abbreviations, and descriptions, except the description of
+    // the target frame, <<12>> and <<34>> are the names and initials of the relevant
+    // celestial bodies, <<34>> being the parent of <<12>>.
+    // Odd: names (Sun^n, Moon^n, Saturn^N, Terre^f, Vénus^F, 火星),
+    // Even: initials if applicable (S, M, S, T, V, 火星).
+
+    #Principia_ReferenceFrameSelector_Name_Target = Cible-<<C:1>>-Orbite
+    #Principia_ReferenceFrameSelector_Name_BodyCentredNonRotating = galiléen centré sur <<A:1>>
```
Comment (pleroy) [inline ksp_plugin_adapter/localization/fr-fr.cfg:52]: inertiel?

File: ksp_plugin_adapter/localization/fr-fr.cfg [hunk 6]
```diff
@@ -0,0 +1,241 @@
...
+    #Principia_ReferenceFrameSelector_Description_BodySurface = <<AC:1>> est fixe.\nSon équateur est horizontal.\nLe réferentiel tourne comme <<A:1>>.
+
+    #Principia_ReferenceFrameSelector_ReferencePlane_Centred = équateur de <<A:1>>  // <<1>> centred body
+    #Principia_ReferenceFrameSelector_ReferencePlane_Secondary_Target = la cible
+    #Principia_ReferenceFrameSelector_ReferencePlane = orbite de <<A:1>> autour de <<A:2>>  // <<1>> secondary body <<2>> centred body
+    #Principia_ReferenceFrameSelector_ToggleButton = Sélection du <<1>> (<<2>>)...  // <<1>> plotting <<2>> selected frame
+    #Principia_ReferenceFrameSelector_Title = Sélection du <<1>> (<<2>>)  // <<1>> plotting <<2>> selected frame
+    // <<1>> is the full name of the reference frame, <<2>> is the central body.
+    #Principia_ReferenceFrameSelector_Tooltip_BodyCentredNonRotating = Réferentiel <<1>> : pour les trajectoires Keplériennes perturbées autour de <<A:2>>.
+    #Principia_ReferenceFrameSelector_Tooltip_BodySurface = Référentiel <<1>> : pour les opérations impliquant la surface de <<A:2>> : atterissages, télédétection, imagerie, télécommunications, etc.
```
Comment (pleroy) [inline ksp_plugin_adapter/localization/fr-fr.cfg:83]: Two colons in sequence are ugly.

File: ksp_plugin_adapter/localization/fr-fr.cfg [hunk 7]
```diff
@@ -0,0 +1,241 @@
...
+    #Principia_ReferenceFrameSelector_Tooltip_BodyCentredParentDirection = Référentiel <<1>> : pour les survols de <<A:2>>, les transferts de ou vers <<A:2>>.
+    #Principia_ReferenceFrameSelector_Tooltip_Target = Référentiel <<1>> : pour les interceptions ou les rendez-vous avec le vaisseau cible.
+    #Principia_ReferenceFrameSelector_Target = Cible : <<1>>
+    #Principia_ReferenceFrameSelector_Pin = Épingler
+
```
Comment (pleroy) [inline ksp_plugin_adapter/localization/fr-fr.cfg:88]: Remove empty line?

File: ksp_plugin_adapter/localization/fr-fr.cfg [hunk 8]
```diff
@@ -0,0 +1,241 @@
...
+    #Principia_OrbitAnalyser_Title = Analyse d’orbite
+    #Principia_OrbitAnalyser_Duration_GroundTrackCycles = \u0020(<<1>> cycles de phasage)  // <<1>> ground_track_cycles.FormatN(0)
+    #Principia_OrbitAnalyser_Duration_Revolutions = <<1>> révolutions sidérales\n<<2>> révolutions nodales<<3>>\n<<4>> révolutions anomalistiques  // <<3>> duration_in_ground_track_cycles
+    #Principia_OrbitAnalyser_Warning_NoElements1 = impossible de déterminer les élements; la durée de la mission est inférieure à une révolution, ou la trajectoire n’est pas liée gravitationellement.
+    #Principia_OrbitAnalyser_Warning_NoElements2 = impossible de déterminer les élements; la durée de la mission est inférieure à une révolution, ou la trajectoire n’est pas liée gravitationellement à <<A:1>>.  // <<1>> primary.NameWithArticle()
+    #Principia_OrbitAnalyser_Warning_NoPrimary = le vaisseau <<1>> n’est pas gravitationellement lié pendant <<2>>  // <<2>>mission_duration.FormatDuration <<1>> vessel name
+    #Principia_OrbitAnalyser_AnalysisDescription = Orbite de <<1>> autour de <<A:2>> pendant <<3>>:\n<<4>>  // <<1>> vessel name <<2>> centre body <<3>> time <<4>> result
```
Comment (pleroy) [inline ksp_plugin_adapter/localization/fr-fr.cfg:98]: Colon.

File: ksp_plugin_adapter/localization/fr-fr.cfg [hunk 9]
```diff
@@ -0,0 +1,241 @@
...
+    // These are celestial strings parameterized on the body.
+    #Principia_OrbitAnalyser_Elements_Periapsis = périastre
+    #Principia_OrbitAnalyser_Elements_Apoapsis = apoastre
+    // In the following three strings, <<1>> is the Periapsis or Apoapsis string
+    // above.
+    #Principia_OrbitAnalyser_Elements_ArgumentOfPeriapsis = Argument du <<1>>
+    #Principia_OrbitAnalyser_Elements_MeanPeriapsisAltitude = Altitude moyenne du <<1>>
+    #Principia_OrbitAnalyser_Elements_MeanApoapsisAltitude = Altitude moyenne de l’<<1>>
+    #Principia_OrbitAnalyser_GroundTrack = Trace au sol
+    #Principia_OrbitAnalyser_Recurrence_CapderouTriple = Phasage: <<1>>
```
Comment (pleroy) [inline ksp_plugin_adapter/localization/fr-fr.cfg:132]: Colon.

File: ksp_plugin_adapter/localization/fr-fr.cfg [hunk 10]
```diff
@@ -0,0 +1,241 @@
...
+
+    // FlightPlanner
+    #Principia_FlightPlan_PlanLength = Longueur
+    #Principia_FlightPlan_ToggleButton = Plan de vol...
+    #Principia_FlightPlan_Title = Plan de vol
+    #Principia_FlightPlan_Create = Créer un plan de vol
+    #Principia_FlightPlan_MaxSteps = Max. pas par segment :
+    #Principia_FlightPlan_TotalΔv = Δv total : <<1>> m/s  // <<1>>: Δv.ToString("0.000").
+    #Principia_FlightPlan_Delete = Supprimer le plan de vol
+    #Principia_FlightPlan_Rebase = Rebaser
```
Comment (pleroy) [inline ksp_plugin_adapter/localization/fr-fr.cfg:161]: Refonder?

File: ksp_plugin_adapter/localization/fr-fr.cfg [hunk 11]
```diff
@@ -0,0 +1,241 @@
...
+    #Principia_FlightPlan_StatusMessage_OutRange3 = la manœuvre nᵒ <<1>>
+    #Principia_FlightPlan_StatusMessage_OutRange4 = la fin du plan de vol
+    #Principia_FlightPlan_StatusMessage_OutRange5 = la manœuvre nᵒ <<1>>
+    #Principia_FlightPlan_StatusMessage_OutRange6 = <<1>>d’ajuster le temps initial ou de réduire la durée de la manœuvre nᵒ <<2>>
+    #Principia_FlightPlan_StatusMessage_OutRange7 = de rallonger le plan de vol ou\u0020
+    #Principia_FlightPlan_StatusMessage_TooShort = le plan de vol est trop court
+    #Principia_FlightPlan_StatusMessage_Increase = d’augmenter la durée du plan de vol
+    #Principia_FlightPlan_StatusMessage_CantRebase = le plan de vol ne peut pas être rebasé pendant l’exécution d’une manœuvre planifiée
+    #Principia_FlightPlan_StatusMessage_WaitFinish = de déplacer la manœuvre ou d’attendre qu’elle soit finie
+    #Principia_FlightPlan_StatusMessage_Last = Les <<1>> dernières manœuvres n’ont pas pu être dessinée parce que <<2>>; essayez <<3>><<4>>.  // <<1>>: anomalous_manœuvres; <<2>>: status_message; <<3>>: remedy_message; <<4>>: #Principia_FlightPlan_StatusMessage_Last2.
```
Comment (pleroy) [inline ksp_plugin_adapter/localization/fr-fr.cfg:193]: Space before semicolon.

File: ksp_plugin_adapter/localization/fr-fr.cfg [hunk 12]
```diff
@@ -0,0 +1,241 @@
...
+    #Principia_BurnEditor_InitialTime = t initial
+    #Principia_BurnEditor_ManœuvringFrame = référentiel de manœuvre
+    #Principia_BurnEditor_MinimizedHeader = <<1>> (<<2>> m/s)  // <<1>>: header, <<2>>: Δv().ToString("0.000").
+    #Principia_BurnEditor_Info_InconsistentFrames = Le référentiel de manœuvre diffère de celui de dessin
+    #Principia_BurnEditor_Delete = Suppr.
+    #Principia_BurnEditor_ActiveEngines = Moteurs actifs
+    #Principia_BurnEditor_ActiveRCS = RCS actif
+    #Principia_BurnEditor_InstantImpulse = Impulsion instantanée
+    #Principia_BurnEditor_InertiallyFixed = Inertiellement fixe
+    #Principia_BurnEditor_TimeBase_StartOfFlightPlan = Base de temps: début du plan de vol
```
Comment (pleroy) [inline ksp_plugin_adapter/localization/fr-fr.cfg:211]: Colon

File: ksp_plugin_adapter/localization/fr-fr.cfg [hunk 13]
```diff
@@ -0,0 +1,241 @@
...
+    #Principia_BurnEditor_TimeBase_EndOfManœuvre = Base de temps: fin de la manœuvre nᵒ <<1>>  // <<1>> index
```
Comment (pleroy) [inline ksp_plugin_adapter/localization/fr-fr.cfg:212]: Same.

File: ksp_plugin_adapter/localization/fr-fr.cfg (hunk 5 above)
Comment (eggrobin) [reply ksp_plugin_adapter/localization/fr-fr.cfg:52]: Galiléen is more common.

File: ksp_plugin_adapter/localization/fr-fr.cfg (hunk 10 above)
Comment (eggrobin) [reply ksp_plugin_adapter/localization/fr-fr.cfg:160]: Rebaser appears to be the translation for the source control sense of _rebase_ which we used for the original.
Alternatively we could be more explicit, there is room (_Recontruire à partir de l’état actuel_ maybe?).

File: ksp_plugin_adapter/localization/fr-fr.cfg [hunk 14]
```diff
@@ -6,47 +6,47 @@ Localization {
...
     #Principia_MainWindow_LoggingSettings_FlushOption = Immédiat
     #Principia_MainWindow_LoggingSettings_RecordJournal = Enregistrer un journal (démarre au chargement)
     #Principia_MainWindow_LoggingSettings_RecordJournalResult = Le journal est <<1>>  // <<1>> ON/OFF
     #Principia_MainWindow_LoggingSettings_JournalingStatus_ON = ACTIF
     #Principia_MainWindow_LoggingSettings_JournalingStatus_OFF = INACTIF
-    #Principia_PredictionSettings_ToleranceLabel = Tolérance:
-    #Principia_PredictionSettings_ToleranceText = <<1>> m  // <<1>>: parameters.length_integration_tolerance.ToString("0.0e0").
-    #Principia_PredictionSettings_Steps = Pas:
+    #Principia_PredictionSettings_ToleranceLabel = Tolérance :
+    #Principia_PredictionSettings_ToleranceText = <<1>> m  // <<1>> : parameters.length_integration_tolerance.ToString("0.0e0").
```
Comment (pleroy) [inline ksp_plugin_adapter/localization/fr-fr.cfg:31]: Nay, don't change the colons in the comments, the comments are in English.  Many places.

File: ksp_plugin_adapter/localization/fr-fr.cfg (hunk 10 above)
Comment (pleroy) [reply ksp_plugin_adapter/localization/fr-fr.cfg:160]: Just "reconstruire" is good I think.

File: ksp_plugin_adapter/localization/fr-fr.cfg [hunk 15]
```diff
@@ -189,7 +189,7 @@ Localization {
     #Principia_FlightPlan_StatusMessage_Increase = d’augmenter la durée du plan de vol
     #Principia_FlightPlan_StatusMessage_CantRebase = le plan de vol ne peut pas être rebasé pendant l’exécution d’une manœuvre planifiée
     #Principia_FlightPlan_StatusMessage_WaitFinish = de déplacer la manœuvre ou d’attendre qu’elle soit finie
-    #Principia_FlightPlan_StatusMessage_Last = Les <<1>> dernières manœuvres n’ont pas pu être dessinées parce que <<2>> ; essayez <<3>><<4>>.  // <<1>> : anomalous_manœuvres ; <<2>> : status_message ; <<3>> : remedy_message ; <<4>> : #Principia_FlightPlan_StatusMessage_Last2.
+    #Principia_FlightPlan_StatusMessage_Last = Les <<1>> dernières manœuvres n’ont pas pu être dessinées parce que <<2>> ; essayez <<3>><<4>>.  // <<1>>: anomalous_manœuvres ; <<2>>: status_message ; <<3>>: remedy_message ; <<4>>: #Principia_FlightPlan_StatusMessage_Last2.
```
Comment (pleroy) [inline ksp_plugin_adapter/localization/fr-fr.cfg:192]: The semicolons in comments, too.

**Pull request merged**